portfoliostylesheet.css

```diff
@@ -25,21 +25,22 @@ body {
 h1{}
 h2 {font-size:1.3rem;
 	color:black;
-	padding-top:15px;
+	padding-top:25px;
 	padding-bottom:15px;
 }
 	
 h3 {
 	text-size:1rem;
 	color:black;
-	padding-bottom:15px;
+	padding-bottom:10px;
 }
 
+
 h4 {}	
```

Comment:
maybe delete empty class selectors to neaten up unless you are in progress of adding styling